packages/file_selector/file_selector/example/ios/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -0,0 +1,122 @@
+{
+  "images" : [
+    {
+      "size" : "20x20",
+      "idiom" : "iphone",
+      "filename" : "Icon-App-20x20@2x.png",
```

Comment:
@stuartmorgan do we usually actually check these binary images in? That seems unnecessary...

Reply:
They are in other ios plugin example folders as well, not to imply they should be.

Reply:
They are small, I wouldn't care so much about the plugin developers needing to download these images (though why?) but also that we're checking in binaries to the git history that will be there forever (without editing history https://stackoverflow.com/questions/10622179/how-to-find-identify-large-commits-in-git-history).  I think binaries are special in this case.  We don't check them into the `flutter/flutter` repo example projects, for example. 

Up to you though, I don't have a strong preference.

Reply:
Filed https://github.com/flutter/flutter/issues/110523

> but also that we're checking in binaries to the git history that will be there forever

Right, it would be good to stop. But incremental copies of the images we already have are basically free because of how git works, so we only pay again when `flutter create` actually changes images.